.github/workflows/ci.yml

```diff
@@ -44,7 +45,7 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-go@v4
```

Comment:
update `actions/setup-go@v4` to `actions/setup-go@v5`

Reply:
updated